Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -101,6 +102,23 @@ void StackingContext::paint(PaintContext& context)
     paint_descendants(context, m_box, StackingContextPaintPhase::FocusAndOverlay);
 }
 
+void StackingContext::paint(PaintContext& context)
+{
+    auto opacity = m_box.computed_values().opacity();
+    if (opacity.has_value() && opacity.value() != 1.0f && opacity.value() != 0.0f) {
```

Comment:
It looks like when opacity == 0, you just fall through into paint_internal(), when you could probably just return immediately.